<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Moustache</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QToolBox" name="toolBox">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="TaskListWidget" name="taskListTools">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>79</width>
          <height>93</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Listes</string>
        </attribute>
       </widget>
       <widget class="TaskWidget" name="taskTools">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>79</width>
          <height>93</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Tâches</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="detailsTolls">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>79</width>
          <height>93</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Détails</string>
        </attribute>
       </widget>
      </widget>
      <widget class="QTreeView" name="tasksView"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>353</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TaskListWidget</class>
   <extends>QWidget</extends>
   <header>CustomWidgets/tasklistwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TaskWidget</class>
   <extends>QWidget</extends>
   <header>CustomWidgets/taskwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
